src/ducks/categories/styles.styl

```diff
@@ -19,25 +19,49 @@
             position absolute
             line-height 24px
 
-    .bnk-cat-figure
-        padding 2rem 0 0
-
     .bnk-cat-chart
-        flex-basis 50%
+        height 182px
         margin-top -100px
+        margin-left auto
```

Comment:
why ?

Reply:
To put it on the right of its flex container